contracts/0.4.24/template/LidoTemplate.sol

```diff
@@ -357,6 +355,28 @@ contract LidoTemplate is IsContract {
         deployState = state;
     }
 
+    function createSimpleDVTApp(
+        uint16[3] _initialSemanticVersion,
+        address _proxy,
+        address _impl,
+        bytes _contentURI
+    ) external onlyOwner {
+        APMRegistry lidoRegistry = deployState.lidoRegistry;
+        Kernel dao = deployState.dao;
+
+        apmRepos.simpleDVT = lidoRegistry.newRepoWithVersion(
+            SIMPLE_DVT_APP_NAME,
+            this,
+            _initialSemanticVersion,
+            _impl,
+            _contentURI
+        );
+
+        bytes32 appId = _getAppId(SIMPLE_DVT_APP_NAME, deployState.lidoRegistryEnsNode);
+        dao.setApp(dao.APP_BASES_NAMESPACE(), appId, _impl);
+        deployState.sdvt = NodeOperatorsRegistry(_proxy);
+    }
```

Comment:
there should be an event maybe (similar to `emit TmplAPMDeployed(address(registry))`)

Reply:
Done in 57ea90e123a532684f21c0a040c63a594517cfda